src/css/story/titling.styl

```diff
@@ -11,9 +11,9 @@
       filter: url('#highlight')
     //color: $focal_text_color
   .contrast-song
-    //color: $contrast_text_color
+    color $contrast_web_color
     //filter: url('#highlight-contrast')
-    background-color $contrast_web_color
+    // background-color $contrast_web_color
```

Comment:
One more unsolicited aesthetic opinion: I think I preferred the look of the colored highlight over coloring the text. Also, it gives it a look that's consistent with the years in the year slider.

(I even kind of liked an earlier version where the main song title always had a focal color highlight.)

Reply:
Was planning on shifting the flood to fill throughout. The reason is less aesthetics and more accessibility. In testing, the flood makes it difficult to read for users with some sight deficiencies.